api/README.md

```diff
@@ -50,3 +50,28 @@ contents:
 
 Once you have, you can relaunch the server, and it should be 
 capable of querying for data from the database.
+
+## Running the Tests
+
+The tests use a local database that's ran in in [docker](https://www.docker.com/).
+If you don't have `docker` installed, you'll need to install it to
+run the tests.
+
+First, start the local database:
+
+```
+./bin/start_local_db.sh
```

Comment:
Is this necessary to also run the API locally? If so, maybe we should add a note about that up above in the API launch instructions on line 15 of the README

Reply:
Nope. The tests start an instance of the API, which connects to this database.

Reply:
D'oh! I'm not sure what I was thinking. Thanks for the clarification!